src/ert/cli/monitor.py

```diff
@@ -24,16 +23,17 @@
 Color = Tuple[int, int, int]
 
 
-def _no_color(
-    s: str,
-    fg: Optional[Color] = None,
-    bg: Optional[Color] = None,
-    style: Optional[Color] = None,
-) -> str:
-    """Alternate color method when no coloring is wanted. Conforms to the
-    signature of ansi_color.color, wherein the first positional argument
-    is the string to be (un-)colored."""
-    return s
+def _no_color(text: str, color: Color) -> str:
```

Comment:
`_colorize` function needs a refactor https://github.com/equinor/ert/blob/129f8ea419c37cab086c24e7e5daab36e0c066ee/src/ert/cli/monitor.py#L59

That is why the tests are currently failing